packages/get-starknet/src/snap.ts

```diff
@@ -37,137 +37,180 @@ export class MetaMaskSnap {
           method: 'starkNet_extractPublicKey',
           params: {
```

Comment:
why here we don't call getSnapParams to set chainId if not set ?

Reply:
good catch